Source/MQTTnet.Server/Internal/MqttConnectedClient.cs

```diff
@@ -227,26 +228,26 @@ async Task HandleIncomingPublishPacket(MqttPublishPacket publishPacket, Cancella
         switch (publishPacket.QualityOfServiceLevel)
         {
             case MqttQualityOfServiceLevel.AtMostOnce:
-            {
-                // Do nothing since QoS 0 has no ACK at all!
-                break;
-            }
+                {
+                    // Do nothing since QoS 0 has no ACK at all!
+                    break;
+                }
             case MqttQualityOfServiceLevel.AtLeastOnce:
-            {
-                var pubAckPacket = MqttPubAckPacketFactory.Create(publishPacket, dispatchApplicationMessageResult);
-                Session.EnqueueControlPacket(new MqttPacketBusItem(pubAckPacket));
-                break;
-            }
+                {
+                    var pubAckPacket = MqttPubAckPacketFactory.Create(publishPacket, dispatchApplicationMessageResult);
+                    Session.EnqueueControlPacket(new MqttPacketBusItem(pubAckPacket));
+                    break;
+                }
             case MqttQualityOfServiceLevel.ExactlyOnce:
-            {
-                var pubRecPacket = MqttPubRecPacketFactory.Create(publishPacket, dispatchApplicationMessageResult);
-                Session.EnqueueControlPacket(new MqttPacketBusItem(pubRecPacket));
-                break;
-            }
+                {
```

Comment:
It seems your code style settings are different and thus the switch statements are reformatted without any reason.